bijou.js

```diff
@@ -323,7 +325,7 @@ let _temp = {
       return "Safari";
     }
     if (isEdge) {
-      return "Edge";
+      return "Edge Legacy";
     }
```

Comment:
The old version was fine, this could be confusing.

Reply:
> The old version was fine, this could be confusing.

@Explosion-Scratch It says in MS docs that the old non-Chromium Edge is called that.

Reply:
Okay! Thanks!